### ISSUE TITLE
Schedule component throws error on drag/drop

### ISSUE THREAD

**Issue by Andreashej** [syncfusion/ej2-angular-ui-components#85]
I am working on a project, where I am using the ejs-schedule component. The problem is that after updating to Angular 8, dragging and dropping events to reschedule no longer works, but throws the following error:
![image](https://user-images.githubusercontent.com/18139331/73245613-eaf00400-41ac-11ea-9076-93abc6870b90.png)

The component is initialised as follows:
`<ejs-schedule #scheduleObj class="col h-100" cssClass="schedule-cell-dimension"
    [selectedDate]="currentCompetition.startdate" [eventDragArea]="droppable-area" [timeScale]="timeScale" [eventSettings]="eventSettings" [group]="group"
    (dragStop)="dragEnd($event)" (popupOpen)="onPopupOpen($event)">(templates&views)</ejs-schedule>`

I initially had problems with referencing the schedule object in my component script, after @ViewChild needed another parameter `{static: false}`, but that was solved by moving the function call to ngAfterViewInit(). But maybe this issue is related, I just didn't find the solution yet.

Hope you can help! :)

**Comment by Andreashej** [comment]
I have narrowed the problem down to only apply to the "Day"-view. The day view is initialised as follows:

`<e-views>
<e-view displayName="Competition" option="Day" [interval]="currentCompetition.getDays()" [startHour]="'07:00'" [endHour]="'23:00'"></e-view>
</e-views>
`

**Comment by VengateshManiraj** [comment]
Dear Customer, 

Greetings from Syncfusion Support. 

Based on your shared details, we have prepared an Angular sample with Angular-cli version as 8 and the drag and drop functionality is working properly at our end. And the sample can be downloaded from the following link, 

Sample: https://www.syncfusion.com/downloads/support/directtrac/general/ze/sample-17616675061157416624 

And for further reference kindly refer the below UG links, 
https://ej2.syncfusion.com/angular/documentation/schedule/getting-started/?no-cache=1 
https://ej2.syncfusion.com/angular/documentation/schedule/appointments/#drag-and-drop-appointments 

Kindly try the above sample and revert us for further assistance.  

Regards,  
Vengatesh

**Comment by Andreashej** [comment]
Hi Vengatest,

Thank you for returning to me.

Your example works fine, and I have discovered that the problem appears when I add grouping and use the "Day" view. Try making the following changes on the app.component.html:
`<ejs-schedule #scheduleObj width='100%' height='650px' [selectedDate]="selectedDate" [eventSettings]="eventSettings" [group]="group">
    <e-resources>
        <e-resource field="group" title="Group" name="groups" [allowMultiple]="false" [dataSource]="groups" idField="id" textField="name"></e-resource>
    </e-resources>
</ejs-schedule>`

And add the following to your app.component.ts:
`  public eventSettings: EventSettingsModel = {
    dataSource: [{
      Id: 1,
      Subject: 'Testing Event',
      StartTime: new Date(2020, 0, 30, 9, 0),
      EndTime: new Date(2020, 0, 30, 11, 0),
      group: 1
    }]
  };

  public group: GroupModel = {
    byDate: true,
    resources: ['groups'],
  };

  public groups = [
    {
      id: 1,
      name: "Group 1"
    },
    {
      id: 2,
      name: "Group2"
    }
  ];`

Now notice the following in the app:
 - Everything works as expected in week-view
 - On Day-view it is no longer possible to drag events up and down to change starting time, but only to drag left and right between groups
 - On Work Week-view notice that when grabbing an event, you can reschedule it, but it also shrinks to be the size of 1 time block while dragging.

Hope you can reproduce.

Best,
Andreas

**Comment by sellakumark** [comment]
@Andreashej - Sorry for the delayed response.

We had validated the reported problem at our end. Currently, drag and drop in day view is working properly towards up and down directions along with different grouping. In work week view, when we reschedule the events, now events grabbing is working properly. We had prepared the sample for both the cases which is available in the below link,

https://stackblitz.com/edit/angular-684tjm-5ovmi2?file=app.component.ts

If you still facing the issue, kindly replicate the defect in above sample  and get back to us for further assistance.
